newIDE/app/src/EventsSheet/EventsTree/InstructionsList.js

```diff
@@ -184,6 +197,7 @@ export default class InstructionsList extends React.Component<Props, State> {
                   style={styles.addButton}
                   className="add-link"
                   onClick={this.onAddNewInstruction}
+                  onMouseDownCapture={this.onQuickAddNewInstruction}
```

Comment:
I've never seen onMouseDownCapture! What is this exactly? Do you think we can use onContextMenu instead? This means that we must remove the existing code in onContextMenu because we're replacing the menu by this.

Reply:
I need to capture the left click to trigger this. Do we have another way already? I need to have another look